locale/en.po

```diff
@@ -1532,9 +1524,9 @@ msgstr "Double click to view in fullscreen"
 msgid "DOUBLE_MEMBRANE_DESCRIPTION"
 msgstr "A membrane with two layers, it has better protection against damage and takes less energy to not deform. However, it slows the cell down some and lowers the rate at which it can absorb resources."
 
-#: ../src/general/MainMenu.tscn:995
+#: ../src/gui_common/dialogs/PermanentlyDismissableDialog.tscn:47
 msgid "DO_NOT_SHOW_AGAIN"
-msgstr "Do not warn about this again"
+msgstr "Don't show this again"
```

Comment:
There's still just the one text... I very intentionally made two variants, one for "warnings" and one for general info dialogs that can be dismissed permanently.

Reply:
If you want you can rename them "DISMISS_WARNING_PERMANENTLY" and "DISMISS_INFORMATION_PERMANENTLY" to clarify the usage of the text.